README.md

```diff
@@ -1,39 +1,82 @@
 
-If nothing found it will try to connect with your hardhat node. On localhost and hardhat nodes it will inject your mnemonic into the frontend so you have a "browser wallet" that can both call and send transactions. NB! Dont ever put a mnemonic with actual value here. We will limit this feature going forward so its more explicit.
+## 🛠 Technology
+- This project was built on Ethereum using [Hardhat](https://hardhat.org/) and [Symfoni](https://github.com/symfoni/hardhat-react-boilerplate)
+- Deposited funds accrue interest through the [AAVE Liquidity Protocol](https://aave.com/)
+- bTokens deployed for each recipient project adhere to the [ERC20 Token Standard](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.3.0/contracts/token/ERC20/IERC20.sol)
 
-```ts
-const config: HardhatUserConfig = {
-  react: {
-    providerPriority: ["web3modal", "hardhat"],
-  },
-};
-```
+### 📖 How it Works (under the hood)
+1. An open-source project is added to Bazaar Finance through the deployment of a new `Vault` and its associated `bToken` contract. For the MVP, this is done by calling `createVault` and `createBToken` in the `VaultFactory` contract. 
+  - The project's `bToken` is deployed first, and then its `Vault` is deployed with the following parameters
+    - `recipient` address
+    - `salary` desired amount the recipient wants to earn each month for working on the project
+    - `token` address of the ERC20 the recipient wishes to be paid in (e.g. DAI)
+    - `aToken` address the interest-accruing version of the `token`, issued through Aave
+    - `bToken` address of the project's bToken, created through `createBToken`
+2. Individuals who wish to contribute the interest on their savings to fund the project can `deposit` a specified `amount` of the projects `token` into its `Vault`.
+3. The deposit gets automatically swapped to the interest-accruing equivalent on Aave, and is stored in the `Vault`.
+4. The depositor gets minted the project's `bToken`, redeemable for underlying funds when they `withdraw` their principal (and any additional interest) from the `Vault`
+5. Interest on deposits accrues to the project's `Vault`. Every 30 days, the accrued interest gets allocated to be `recipient`, available to be withdrawn.
+6. If the interest accrued from the sum of deposited funds exceeds the recipient's desired monthly `salary` for that month, additional interest earned for the remainder of that month gets allocated to depositors.
+7. Recipients can `withdraw` a specified `amount` at any time.
+8. Depositors can `withdraw` their principal (and any additional interest) at any time.
+
+Note: The `Vault` has sole access to `mint` and `burn` on its associated `bToken`, as these functions need only be called during `deposit` and `withdraw`
```

Comment:
do we wanna mention additional note that, for current implementation:

1. `withdraw` will withdraw their entire balance and 2. user
2.  users aren't allow to make transfers of bTokens?

Reply:
good points! added 👍🏼